x-pack/solutions/security/plugins/security_solution/public/asset_inventory/context.tsx

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { createContext, memo, useContext } from 'react';
+import type { UseAssetInventory } from './hooks/use_asset_inventory';
+import { useAssetInventory } from './hooks/use_asset_inventory';
+
+const defaultContextValue: UseAssetInventory = {
+  isLoading: true,
+  refetchStatusFn: () => {},
+};
```

Comment:
I wouldn't create a context that just shares information from an already sharable hook. i think its an overkill. keep in mind reactquery already manages state in the background so if we call for a refetch, all hooks that used that data will re-render.